packages/query/src/__tests__/query.test.ts

```diff
@@ -757,6 +759,10 @@ describe('query', () => {
       )
     })
 
+    await new Promise((resolve) => {
```

Comment:
Since query call for first time after callback 1ms. Probable we need to remove it, and after optimization it become not called expected by test two times.